models/agol.js

```diff
@@ -990,6 +990,7 @@ var AGOL = function (koop) {
           if (err) agol.log.error(err)
           try {
             error = JSON.parse(value)
+            if (error.type === 'db') return removeJob(job)
```

Comment:
it looks like the error never gets recorded at this point. is that something we want to capture @dmfenton?